lib/ansible/executor/powershell/module_manifest.py

```diff
@@ -294,7 +294,7 @@ def _create_powershell_wrapper(b_module_data, module_path, module_args,
         exec_manifest["actions"].insert(0, 'async_wrapper')
         exec_manifest["async_jid"] = str(random.randint(0, 999999999999))
         exec_manifest["async_timeout_sec"] = async_timeout
-        exec_manifest["async_named_pipe_timeout"] = C.config.get_config_value('DEFAULT_WIN_ASYNC_STARTUP_TIMEOUT')
+        exec_manifest["win_async_startup_timeout"] = C.DEFAULT_WIN_ASYNC_STARTUP_TIMEOUT
```

Comment:
I would still keep this as `async_startup_timeout` or `async_startup_timeout_sec`. This value does not have to align with the ini section in the docs. It just needs to align with what `async_wrapper.ps1` is looking for in the payload.

The `ini` section in the config yaml just states how to define that config option in `ansible.cfg`. As it stands right now you can define it like

```
[defaults]
win_async_startup_timeout = 10
```